projects/data-atlas-app/src/app/app.component.html

```diff
@@ -30,11 +30,17 @@
         <wave-small-time-interaction class="mat-elevation-z4"></wave-small-time-interaction>
         <wave-zoom-handles (zoomIn)="map.zoomIn()" (zoomOut)="map.zoomOut()" class="mat-elevation-z4"></wave-zoom-handles>
     </div>
-    <button mat-flat-button id="analysisButton" class="mat-elevation-z4" *ngIf="!(analysisVisible$ | async)" (click)="showAnalysis()">
+    <button
+        mat-flat-button
+        id="analysisButton"
+        class="mat-elevation-z4"
+        *ngIf="(analysisVisible$ | async) === false"
+        (click)="showAnalysis()"
+    >
         <mat-icon svgIcon="cogs"></mat-icon>
         Analysis
     </button>
-    <mat-card id="analysisCard" *ngIf="analysisVisible$ | async"> Analysis </mat-card>
+    <mat-card id="analysisCard" *ngIf="analysisVisible$ | async"> Analysis</mat-card>
```

Comment:
so trailing spaces at the end is ok but not in front?

Reply:
I'm not sure about this one but spaces in HTML have meanings and you can erase some (e.g. multiple consecutive spaces) and some you can't without changing the behavior :shrug:.